### ISSUE TITLE
Atomic Use For "suspended"

### ISSUE THREAD

**Issue by Rincewind34** [ffAudio/ff_meters#20]
I think the bool suspended in LevelMeterSource would make a great use of an atomic boolean. This way (and I think it is intended that way?) the UI would be able to suspend a meter source since it nows when it is not showing the level meter. The audio thread doesn't even have to call measureBlock if it wants to suspend the level meter.

Let me know what you think / if I should create a pull request.

**Comment by ffAudio** [comment]
You are right, it should be atomic. I am just thinking about it though, maybe it shouldn't be a flag but a ref counter how many UI objects are looking at it. There could be any number of LevelMeters looking at it and if one is destroyed it must not reset the flag if other meters still look at it.
I will add some counter...
Thank you for the suggestion.

**Comment by Rincewind34** [comment]
On a related note: the newDataFlag should also become atomic
